prometheus/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
```

Comment:
Maybe I'm lazy, but why not use the Prometheus rust [crate](https://crates.io/crates/prometheus)?